extension/src/testOutputChannel.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import { Disposable, OutputChannel, window } from 'vscode';
+
+class TestOutputChannel implements Disposable {
+    private readonly channel: OutputChannel = window.createOutputChannel('Java Test Runner');
+
+    public info(info: string): void {
+        this.appendLine(`[INFO] ${info}`);
+    }
+
+    public error(error: string): void {
```

Comment:
Might take `error(message: string, error: Error)` into consideration for unified output format.